gno.land/pkg/integration/testing_integration.go

```diff
@@ -12,7 +12,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/gnolang/gno/gnovm/pkg/gnoroot"
```

Comment:
```suggestion
	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
```

Reply:
and later, we'll move in gno.land/pkg, but right now we have a diamond dependency issue; so let's keep it in gnovm for now